### PR TITLE
feat(home): create feedback panel

### PR DESCRIPTION
**Pull request by priyankajayaswal**
Fixes: #56 

Old version without material-ui
image

Latest version with material-ui
image

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useEffect } from 'react'
 import { useRouter } from 'next/router'
 
 import * as ga from './../lib/ga'
+import "./../styles/app.css"
```
Comment (souvikbasu) [inline pages/_app.tsx:8]: do we need to use custom css? Can the styling be done using tailwind classes?

File: pages/feedback.table.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import 'tailwindcss/tailwind.css';
```
Comment (souvikbasu) [inline pages/feedback.table.tsx:1]: This component should ideally go into the components folder and not pages

File: pages/feedback.table.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  });
+
+  const set = (name: string) => {
+    return ({ target: { value } }) => {
+      setValues(oldValues => ({ ...oldValues, [name]: value }));
+    }
+  };
+  
+  const saveFormData = async () => {
+    const urlLink = `https://wt-5c9a92f46a5c0d255deef1b49b7375ca-0.run.webtask.io/wallora-feedback/?firstName=${values.firstName}&lastName=${values.lastName}&email=${values.email}&feedback=${values.feedbackText}`
```
Comment (souvikbasu) [inline pages/feedback.table.tsx:17]: can we move the url template to env config file? That way we can provide different URL for different env

File: pages/feedback.table.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    const response = await fetch(urlLink, {
+      method: 'GET'
+    });
+
+    if (response.status !== 200) {
+      throw new Error(`Request failed: ${response.status}`); 
+    }
+  }
+
+  const onSubmit = async (event: any) => {
```
Comment (souvikbasu) [inline pages/feedback.table.tsx:28]: set type of event instead of any

File: pages/feedback.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import React from 'react';
+import { AppProps } from 'next/app';
+import FeedBackTable from './feedback.table';
+import useToggle from '../lib/toggle.helper';
+
+
+function FeedBack({ Component, pageProps }: AppProps) {
+  const [isOn, toggleIsOn] = useToggle(false);
+  
+  if (isOn)
```
Comment (souvikbasu) [inline pages/feedback.tsx:11]: use a ternary operator to call both the render methods

File: styles/app.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  width: 394px;
+}
+
+.register-textbox.narrow {
+  margin-right: 10px;
+  width: 192px;
+}
+
+
+button, input, optgroup, select, textarea {
```
Comment (souvikbasu) [inline styles/app.css:15]: let's integrate material UI instead of define our own UI controls like buttons

File: styles/app.css [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  line-height: 1.42857143;
+  margin: 0;
+  padding: 2px 8px;
+}
+
+table td, table td * {
+  vertical-align: top;
+}
+
+.feedback-panel {
```
Comment (souvikbasu) [inline styles/app.css:28]: use tailwind for all the styling. USe custom CSS only if not possible to di using tailwind

**Review by souvikbasu — CHANGES_REQUESTED** [review]

**Review by souvikbasu — CHANGES_REQUESTED** [review]

File: pages/feedback.table.tsx (hunk 3 above)
Comment (priyankajayaswal1) [reply pages/feedback.table.tsx]: updated

File: lib/toggle.helper.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import {useState , useCallback} from 'react';
```
[the remark this answers is not in the document]
Comment (priyankajayaswal1) [reply lib/toggle.helper.tsx:1]: updated

**Review by souvikbasu — CHANGES_REQUESTED** [review]
please squash all commits into one and add commit message as per our format

File: components/Feedback/index.tsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div className="fixed text-sm duration-200 top-80" style={buttonStyle}>
+      <div
+        role="button"
+        onClick={toggleIsOn}
+        onKeyPress={toggleIsOn}
+        tabIndex={0}
+        className="text-white text-center text-fb bg-orange w-40 origin-left p-2 cursor-pointer h-14 -mt-8 -ml-7 -mb-56 -mr-30 shadow-fb transform -rotate-90"
+      >
+        GET IN TOUCH
```
Comment (souvikbasu) [inline components/Feedback/index.tsx:19]: use uppercase tailwind class to capitalize. Keep the text as "Get in touch"

**Review by souvikbasu — CHANGES_REQUESTED** [review]

**Review by souvikbasu — APPROVED** [review]

**Pull request merged**